src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+.calendar {
+  $dayCellSize: 100px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  margin-top: 50px;
+  gap: 1px;
+  width: 7 * $dayCellSize + 6px;
+  padding: 10px;
+  border: 10px solid #262626;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)